backends-velox/src/test/resources/tpch-approved-plan/v1-ras/spark32/3.txt

```diff
@@ -1,56 +1,64 @@
 == Physical Plan ==
-AdaptiveSparkPlan (59)
+AdaptiveSparkPlan (67)
 +- == Final Plan ==
-   VeloxColumnarToRowExec (39)
-   +- TakeOrderedAndProjectExecTransformer (38)
-      +- ^ ProjectExecTransformer (36)
-         +- ^ RegularHashAggregateExecTransformer (35)
-            +- ^ RegularHashAggregateExecTransformer (34)
-               +- ^ ProjectExecTransformer (33)
-                  +- ^ ShuffledHashJoinExecTransformer Inner BuildLeft (32)
-                     :- ^ InputIteratorTransformer (23)
-                     :  +- ShuffleQueryStage (21)
-                     :     +- ColumnarExchange (20)
-                     :        +- ^ ProjectExecTransformer (18)
-                     :           +- ^ ShuffledHashJoinExecTransformer Inner BuildLeft (17)
-                     :              :- ^ InputIteratorTransformer (8)
-                     :              :  +- ShuffleQueryStage (6)
-                     :              :     +- ColumnarExchange (5)
-                     :              :        +- ^ ProjectExecTransformer (3)
-                     :              :           +- ^ NoopFilter (2)
-                     :              :              +- ^ Scan parquet (1)
-                     :              +- ^ InputIteratorTransformer (16)
-                     :                 +- ShuffleQueryStage (14)
-                     :                    +- ColumnarExchange (13)
-                     :                       +- ^ ProjectExecTransformer (11)
-                     :                          +- ^ NoopFilter (10)
-                     :                             +- ^ Scan parquet (9)
-                     +- ^ InputIteratorTransformer (31)
-                        +- ShuffleQueryStage (29)
-                           +- ColumnarExchange (28)
-                              +- ^ ProjectExecTransformer (26)
-                                 +- ^ NoopFilter (25)
-                                    +- ^ Scan parquet (24)
+   VeloxColumnarToRowExec (43)
+   +- TakeOrderedAndProjectExecTransformer (42)
+      +- ^ ProjectExecTransformer (40)
+         +- ^ RegularHashAggregateExecTransformer (39)
+            +- ^ RegularHashAggregateExecTransformer (38)
+               +- ^ ProjectExecTransformer (37)
+                  +- ^ ShuffledHashJoinExecTransformer Inner BuildLeft (36)
+                     :- ^ SortExecTransformer (26)
```

Comment:
Can't drop `SortExecTransformer` when RAS enabled is a known issue, #6011 @zhztheplayer